wallet/src/de/schildbach/wallet/ui/BaseMenuActivity.kt

```diff
@@ -31,7 +31,7 @@ import de.schildbach.wallet_test.R
  * @author Samuel Barbosa
  */
 @SuppressLint("Registered")
-abstract class BaseMenuActivity : AppCompatActivity() {
+abstract class BaseMenuActivity : AbstractWalletActivity() {
```

Comment:
@sambarboza FYI 
It's required by Backup wallet flow and View recovery phrase flow